lib/v2/reuters/migration_prompt.js

```diff
@@ -0,0 +1,3 @@
+module.exports = async (ctx) => {
+    ctx.state.data = await {};
```

Comment:
```suggestion
    throw new Error('The route has been deprecated. Please refer to https://docs.rsshub.app/traditional-media.html#lu-tou-she for more details.');
```

Reply:
Please also edit your PR description to let the PR test workflow test the migration prompt.